File: 250517/더 긴 문자열/longer-string.js
const fs = require("fs");

let input = fs.readFileSync(0).toString().trim().split(" ");

console.log(input[0].length > input[1].length ?
     `${input[0]} ${input[0].length}` : input[1].length > input[0].length ? 
     `${input[1]} ${input[1].length}` : "same")

File: 260320/더 긴 문자열/longer-string.py
import sys

input = sys.stdin.readline

# 선언부

# 구현부
s1, s2 = input().rstrip().split()
print('same' if len(s1) == len(s2) else f'{s1} {len(s1)}' if len(s1) > len(s2) else f'{s2} {len(s2)}')